### PR TITLE
modules: Test module enable

### PR DESCRIPTION
**Pull request by pkratoch**

### CODE REVIEW

File: include/libdnf5/base/base.hpp [hunk 1]
```diff
@@ -125,6 +127,7 @@ class Base {
     friend class libdnf5::module::ModuleSack;
     friend class libdnf5::repo::RepoSack;
     friend class libdnf5::repo::SolvRepo;
+    friend class ::ModuleTest;
```
Comment (j-mracek) [inline include/libdnf5/base/base.hpp:130]: This is not the best approach. May I ask you to look at `base_test_case.cpp` for inspiration.

```
// Accessor of private Base::p_impl, see private_accessor.hpp
create_private_getter_template;
create_getter(priv_impl, &libdnf5::Base::p_impl);
```

Comment (pkratoch) [reply include/libdnf5/base/base.hpp]: Wow, ok, I didn't know the private accessor. I changed it, tough I don't understand how this thing works. Maybe we can discuss it later.

**Comment by pkratoch** [comment]
Rebased.

**Comment by j-mracek** [comment]
LGTM

**Pull request merged**